hangman.py

```diff
@@ -0,0 +1,256 @@
+    '''
+    # FILL IN YOUR CODE HERE AND DELETE "pass"
+    pass
+
+
+def get_guessed_word(secret_word, letters_guessed):
+    '''
+    secret_word: string, the word the user is guessing
+    letters_guessed: list (of letters), which letters have been guessed so far
+    returns: string, comprised of letters, underscores (_), and spaces that represents
+      which letters in secret_word have been guessed so far.
+    '''
+    # FILL IN YOUR CODE HERE AND DELETE "pass"
+    pass
+
+
+def get_available_letters(letters_guessed):
+    '''
+    letters_guessed: list (of letters), which letters have been guessed so far
+    returns: string (of letters), comprised of letters that represents which letters have not
+      yet been guessed.
+    '''
+    # FILL IN YOUR CODE HERE AND DELETE "pass"
+    pass
+
+
+def warnings(value_mistake, warnings_point, guesses_point, secret_word, letters_guessed):
+    """
+    Subtract warnings point and output name of error
+    """
+    if warnings_point > 0:
+        warnings_point -= 1
+        print(f"Oops! {value_mistake}", "\nYou have", warnings_point, "warnings left:",
+              get_guessed_word(secret_word, letters_guessed))
+    else:
+        #if user does not have a warnings points, he\she lose one guesses point
+        guesses_point -= 1
+        print("Oops! You have no warnings left, so you lose one guess:", get_guessed_word(secret_word, letters_guessed))
+
+    return warnings_point, guesses_point
+
+
+def print_hints_warning(list_word):
+    pass
+
+
+def letter_in_word(secret_word, letter, guesses_point, letters_guessed):
+    """
+    Check if letter is word, else subtracts point
+    """
+    letters_guessed.add(letter)
+    if letter in secret_word:
+        print("Good guess:", get_guessed_word(secret_word, letters_guessed))
+    else:
+        if letter in LIST_VOWELS:
+            guesses_point -= 2
+        else:
+            guesses_point -= 1
+        print("Oops! That letter is not in my word:", get_guessed_word(secret_word, letters_guessed))
+
+    return letters_guessed, guesses_point
+
+
+count_star = 0
+
+
+def print_classic_warning(warnings_point, guesses_point, secret_word, letters_guessed):
+    global count_star
+    if count_star != 0:
+        warnings_point, guesses_point = warnings("you are playing classic hangman", warnings_point,
+                                                 guesses_point, secret_word, letters_guessed)
+    else:
+        print("Oops, you are playing classic hangman:", get_guessed_word(secret_word, letters_guessed))
+        count_star += 1
+
+    return warnings_point, guesses_point
+
+
+def check_letter(letter, warnings_point, guesses_point, secret_word, letters_guessed):
+    '''
+    This function check if letter is correct. So letter is correct, if it is acsii_letters, len(letter) is 1 and
+    letter not in letters_guessed. Additionally, it check that letter is or is not a GAME_WITH_HINTS("*")
+    '''
+
+    if letter == GAME_WITH_HINTS:
```

Comment:
It looks strange that letter can be equal to game. Could you rename the constant? e.g. HINTS_SYMBOL